operator/cmd/operator/app/operator.go

```diff
@@ -110,6 +114,28 @@ func Run(ctx context.Context, o *options.Options) error {
 		return err
 	}
 
+	if err := manager.AddReadyzCheck("readyz", healthz.Ping); err != nil {
+		klog.Errorf("Unable to set up ready check: %v", err)
+		return err
+	}
+
+	// Unregister default NewGoCollector
+	ctrlmetrics.Registry.Unregister(collectors.NewGoCollector())
+
+	ctrlmetrics.Registry.MustRegister(
+		collectors.NewGoCollector(
+			collectors.WithGoCollectorRuntimeMetrics(
+				collectors.MetricsGC,
+				collectors.MetricsScheduler,
+				collectors.MetricsMemory,
+				collectors.GoRuntimeMetricsRule{Matcher: regexp.MustCompile(`^/sync/.*`)},
+			),
+		),
+	)
```

Comment:
Is the intention here to select only useful metrics? Unregister the default ones and then add the needed one ?

Reply:
Thanks @RainbowMango 
In `collectors.NewGoCollector()` just basic metrics(e.g. "go_goroutines" and "go_info",  etc).  Add `MetricsGC`, `MetricsScheduler`,  `MetricsMemory` and `GoRuntimeMetricsRule` more info, and it includes basic metrics.

Reply:
Get it. Thanks. I never looked into these metrics before, I will test it on my side.